### PR TITLE
add the VP samples README.

### PR DESCRIPTION
**Pull request by kelly-du**
Signed-off-by: kelly-du <kelly.du@intel.com>

### CODE REVIEW

File: videoprocess/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Video Processing feature test suites
+
+
+## Introduction
+
+This folder collect a set of VP examples to exercise VA-API in
+accordance with the libva project. 
+vaapp is the basic app which you can use to check several features(CiSC,scaling,
```
Comment (jackie-guo) [inline videoprocess/README.md]: Typo: CSC instead of CiSC.

File: videoprocess/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## How to run the sample
+
+1. The app para should be defined in the corresponding *.cfg file. Each app will have
+one related *.cfg.template for your reference,and the detailed para meaning is described in
+the file. you can create your *.cfg according to your usage.
+
+2. Copy the app and cfg file to the target machine and run.
+```
+$ ./vavpp process.cfg
```
Comment (jackie-guo) [inline videoprocess/README.md]: How about adding command line for 1:N and USERPTR?

File: videoprocess/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Video Processing feature test suites
```
Comment (jackie-guo) [inline videoprocess/README.md]: Please check with Carl to make sure this format is suitable for open source.

File: videoprocess/README.md (hunk 1 above)
Comment (kelly-du) [reply videoprocess/README.md]: done

File: videoprocess/README.md (hunk 2 above)
Comment (kelly-du) [reply videoprocess/README.md]: As maybe the PR be merged before usrptr sample, so I write the vavapp for sample use. anyway, I change to usrptr usage as the command line.

**Pull request merged**